dbt_copilot_helper/commands/codebase.py

```diff
@@ -0,0 +1,74 @@
+import stat
+import subprocess
+from pathlib import Path
+
+import click
+import requests
+import yaml
+
+from dbt_copilot_helper.utils.click import ClickDocOptGroup
+from dbt_copilot_helper.utils.files import mkfile
+from dbt_copilot_helper.utils.template import setup_templates
+from dbt_copilot_helper.utils.versioning import (
+    check_copilot_helper_version_needs_update,
+)
+
+
+@click.group(chain=True, cls=ClickDocOptGroup)
+def codebase():
+    """Codebase commands."""
+    check_copilot_helper_version_needs_update()
+
+
+@codebase.command()
+def prepare():
+    templates = setup_templates()
+
+    repository = (
+        subprocess.run(["git", "rev-parse", "--show-toplevel"], capture_output=True, text=True)
+        .stdout.split("/")[-1]
+        .strip()
+    )
+
+    if "deploy" in repository:
```

Comment:
Is it just better to look for a copilot directory in here rather than depending on the project name? Or at least look specifically for a "-deploy" suffix rather than just "deploy" anywhere in the name?

Reply:
I think just the `copilot` dir check would be an idea here as the repo name could be something other than expected.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/206/commits/133a3ebc2000c75428931e4b8840373b5b6a099c.

Reply:
Didn't see @acodeninja comment before... I think that's a great point - resolved in https://github.com/uktrade/copilot-tools/pull/206/commits/f59ac44c540acb453cd0dfd330fefa9c726ecca3.